android/app/src/main/java/com/mattermost/rnbeta/RNPasteableEditTextOnPasteListener.java

```diff
@@ -138,13 +139,14 @@ private String moveToImagesCache(String src, String fileName) {
         String dest = cacheFolder + fileName;
         File folder = new File(cacheFolder);
 
-
         try {
             if (!folder.exists()) {
                 folder.mkdirs();
             }
 
             Files.move(Paths.get(src), Paths.get(dest));
+        } catch (FileAlreadyExistsException fileError) {
+            // No nothing and return dest path
```

Comment:
added as if the file already existed this was returning an error instead of the file path